lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -107,44 +122,20 @@ def self.process_network_mask(network)
       network
     end
 
-    def self.depth(a)
-      return 0 unless a.is_a?(Array)
-      1 + depth(a[0])
-    end
-
     # Helper to build a hash of add/remove commands for a nested array.
     # Useful for network, redistribute, etc.
     #   should: an array of expected cmds (manifest/recipe)
     #  current: an array of existing cmds on the device
     def self.delta_add_remove(should, current=[])
```

Comment:
I recall I removed the return in 'return 1 + depth' as rubocop flagged it. That was the only change I recall. but the below looks wrong to me as well.